packages/fresh/src/test_utils.ts

```diff
@@ -137,10 +137,15 @@ export async function withTmpDir(
   return {
     dir,
     async [Symbol.asyncDispose]() {
+      // Skip pointless cleanup in CI, speed up tests
+      if (Deno.env.get("CI") === "true") return;
+
       try {
         await Deno.remove(dir, { recursive: true });
       } catch {
-        // Ignore errors Files in tmp will be cleaned up by the OS
+        // Temp files are not cleaned up automatically on Windows
+        // deno-lint-ignore no-console
+        console.warn(`Failed to clean up temp dir: "${dir}"`);
```

Comment:
Can be helpful when running locally, because if failing to remove the dirs it may spam the `<user>\AppData\Local\Temp` dir with new folders on WIndows that are not cleaned up if running consecutive tests.